.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -37,6 +37,16 @@ assignees: ''
 2. ...
 3. ...
 
+<!--
+     Please tell us on which platform(s) the problem occurs (Android / iOS / Web / Desktop)
+     Which OS version is the test system running?
```

Comment:
For web we want the browser and browser version, not the OS version, presumably.